Makefile

```diff
@@ -49,7 +49,10 @@ default:
 shell:
 	@$(RUN) /bin/bash
 
-.PHONY: default shell
+rust-version:
```

Comment:
this file is copied from common-files repo and will be reverted. Only makefile.core.mk should be used. 

you could just list "common/scripts/run.sh rustc --version" or similar

Reply:
After test, run in core.md would raise an error since no BUILD_WITH_CONTAINER value enabled, so write it in the overrides.mk.

Reply:
I believe a simple:
```
rust-version:
	rustc -vV
```
should have worked in the Makefile.core.mak but this works also.

Reply:
> I believe a simple:
> 
> ```
> rust-version:
> 	rustc -vV
> ```
> 
> should have worked in the Makefile.core.mk but this works also.

This need to be run in the dev container.

Reply:
`make rust-version` would start the build-tools container and then run the same command inside the container. Assuming one isn't overriding the config to not run inside the build container. Or maybe I'm misunderstanding your setup.

Reply:
`/common/scripts/run.sh /usr/bin/rustc -vV` will work regardless of BUILD_WITH_CONTAINER, which is what we want.

But @ericvn is correct - @daixiang0 can you just move this target to `Makefile.core.mk` with the other targets, the command you have works just as well there.

Reply:
But beware as running rustc outside the container will tell you your local version of rust (if you have it installed, else an error), and not the one in the build-tools container, which is what the doc says.

Reply:
> But beware as running rustc outside the container will tell you your local version of rust (if you have it installed, else an error), and not the one in the build-tools container, which is what the doc says `You can determine the version that the `build-tools` image uses...`

Yep with `run.sh` it uses the build container regardless of whether I specify `BUILD_WITH_CONTAINER=0|1` on the CLI, so we should be good on that front.

Reply:
Gotcha. So we want this to work from the command line irrespective of if the developer has BUILD_WITH_CONTAINER set to 0? That does make the problem trickier. What's the probability that people are running with BUILD_WITH_CONTAINER=0? It's probably higher than I think..

From my testing, what is in this PR right now (calling run... in the Makefile) would work if BUILD_WITH_CONTAINER=1, but will fail with BUILD_WITH_CONTAINER=0. Also if someone is inside a `make shell`, a `make rust-version` will fail as well.

My suggestion works in the shell and locally except for `BUILD_WITH_CONTAINER=0 make rust-version` as that will try and show the locally installed rust version.

Would need to try some more experimentation to find something that might work everywhere.

Reply:
Yeah I apologize - I noticed that the `build-tools` image was pulled either way, but you're correct, whether the actual command runs  is contingent on that variable.

```
rust-version:
	rustc -vV
```

is enough for this, IMO. It will work inside and outside (unless someone overrides defaults) If someone is explicitly setting `BUILD_WITH_CONTAINER=0` (not the default) - they are already on their own as far as build tooling.

We can fence the run with `BUILD_WITH_CONTAINER` checks but that's excessive.

Reply:
@bleggett Done.